scripts/test-e2e-standalone.sh

```diff
@@ -16,6 +16,15 @@ TMPDIR=$(mktemp -d)
 # ARTIFACT_DIR is defined when running in a prow job
 ARTIFACT_DIR=${ARTIFACT_DIR:-$TMPDIR}
 
+# Downloading index store
+export RAG_INDEX="https://github.com/ilan-pinto/lightspeed-rag-documents/releases/latest/download/local.zip"
+export RAG_INDEX_DIR="$ARTIFACT_DIR/vector-db/ocp-product-docs"
```

Comment:
let's use a separate tmpdir for the RAG content, we don't want it in ARTIFACTS_DIR because we don't want it collected as part of the job artifacts (it's a waste of storage)